src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,92 @@
+package duke.storage;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Storage {
+    private String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Loads the user's stored list.
+     * @return An arraylist containing the user's stored list.
+     * @throws DukeException if there is an issue with file access.
+     */
+    public ArrayList<Task> load() throws DukeException {
+        BufferedReader reader;
+        ArrayList<Task> storedTasks = new ArrayList<>(100);
+
+        try {
+            String line;
+            reader = new BufferedReader(new FileReader(filepath));
+
+            while ((line = reader.readLine()) != null) {
+                Task task = stringToTask(line);
+                storedTasks.add(task);
+            }
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+            throw new DukeException("IO Exception");
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        } // End trycatch.
+        return storedTasks;
+    } // End method.
+
+    /**
+     * Converts the stored string to a task.
+     * @param str A string representing a task.
+     * @return A task.
+     * @throws DukeException if the stored data is corrupted.
+     */
+    public Task stringToTask(String str) throws DukeException {
+        String[] split = str.split("\\|");
+        try {
+            boolean isDone = (split[0].trim()).equals("1");
+            switch (split[1].trim()) {
+            case "T": // Todo.
```

Comment:
Since you used ENUM for the commands, maybe you can do the same for these letters? 
It's just a suggestion. Your codes look totally fine! 👍